generators/generator-bot-adaptive/platforms.js

```diff
@@ -4,4 +4,7 @@
 const dotnet = 'dotnet';
 const js = 'js';
```

Comment:
It might be a slightly more involved refactor, but maybe a platform should be an object like this:

```javascript
const js = {
  name: 'js',
  defaultSdkVersion: '4.13.4-preview',
}
```